### PR TITLE
Add dry run mode

### PR DESCRIPTION
**Pull request by czerasz-flaconi**
When running `ansible-playbook` with `--check` flag, run Helm with `--dry-run` flag.

### CODE REVIEW

File: tasks/run.yml [hunk 1]
```diff
@@ -51,22 +51,60 @@
...
+    - (item.skipped is not defined) or (not item.skipped)
 
 - name: Update chart (if release exist already)
   command: |
     helm upgrade
       {% if helm_kubectl_context is defined and helm_kubectl_context %}
       --kube-context {{ helm_kubectl_context }}
       {% endif %}
+      {% if ansible_check_mode %}
+      --dry-run
```
Comment (cytopia) [inline tasks/run.yml]: I guess this will then be always reported as changed or?

Comment (czerasz-flaconi) [reply tasks/run.yml]: Yes. Unfortunately I couldn't find a solution to keep this unchanged.

Comment (czerasz-flaconi) [reply tasks/run.yml]: Refactored

File: tasks/run-dry.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- name: Show dry mode results
+  debug:
+    msg:
+      cmd: "{{ item.cmd | join(' ') }}"
+      stdout: "{{ item.stdout }}"
+      stderr: "{{ item.stderr }}"
+      return_code: "{{ item.rc }}"
+  with_items: "{{ _helm_install_chart.results }}"
+  # Run only for items which were not skipped
+  when: (item.skipped is not defined) or (not item.skipped)
```
Comment (cytopia) [inline tasks/run-dry.yml:29]: Is this still required

File: tasks/run-dry.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- name: Show dry mode results
+  debug:
+    msg:
+      cmd: "{{ item.cmd | join(' ') }}"
+      stdout: "{{ item.stdout }}"
+      stderr: "{{ item.stderr }}"
+      return_code: "{{ item.rc }}"
+  with_items: "{{ _helm_upgrade_chart.results }}"
+  # Run only for items which were not skipped
+  when: (item.skipped is not defined) or (not item.skipped)
```
Comment (cytopia) [inline tasks/run-dry.yml:57]: Is this still required?

**Review by cytopia — APPROVED** [review]
Excellent work :+1: 
Is this going to be a minor or patch level tag?

**Pull request merged**